<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settleWindow</class>
 <widget class="QMainWindow" name="settleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>订单结算中心</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image/image.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 15px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 2px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 2px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom:2px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 2px solid #3ebaba;

background: aqua; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>订 单 结 算 中 心</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 15px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 2px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 2px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 2px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 2px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>当前账号：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商品名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="shoppingnameEdit_">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton { /* 按钮美化 */
 width: 50px; /* 宽度 */
 height: 50px; /* 高度 */
 border-width:3px; /* 边框宽度 */
 border-radius: 10px; /* 边框半径 */
 border-style: outset; 
 border-color:#00aaff;
 background: aqua; /* 背景颜色 1E90FF*/
 color: black; /* 字体颜色 */

}
QPushButton:hover { /*5599FF 鼠标移入按钮范围时改变颜色 */
 background: #00d9d9;
 border-style: inset;
 color: black; /* 字体颜色 */
}</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QListWidget" name="shoppinglistWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>481</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>序号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商品名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>单价</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton { /* 按钮美化 */
 width: 50px; /* 宽度 */
 height: 50px; /* 高度 */
 border-width:3px; /* 边框宽度 */
 border-radius: 10px; /* 边框半径 */
 border-style: outset; 
 border-color:#00aaff;
 background: aqua; /* 背景颜色 1E90FF*/
 color: black; /* 字体颜色 */

}
QPushButton:hover { /*5599FF 鼠标移入按钮范围时改变颜色 */
 background: #00d9d9;
 border-style: inset;
 color: black; /* 字体颜色 */
}</string>
    </property>
    <property name="text">
     <string>库存商品对照表</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="stocklistWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发票</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="invoicelistWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="settleButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton { /* 按钮美化 */
 width: 50px; /* 宽度 */
 height: 50px; /* 高度 */
 border-width:3px; /* 边框宽度 */
 border-radius: 15px; /* 边框半径 */
 border-style: outset; 
 border-color:#00aaff;
 background: aqua; /* 背景颜色 1E90FF*/
 color: black; /* 字体颜色 */

}
QPushButton:hover { /*5599FF 鼠标移入按钮范围时改变颜色 */
 background: #00d9d9;
 border-style: inset;
 color: black; /* 字体颜色 */
}</string>
    </property>
    <property name="text">
     <string>结算</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>应收金额：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="moneylabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>序号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商品名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>库存</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商品名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>总计金额：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="moneylabel1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>注：点击右上角&quot;x&quot;可以返回上一界面</string>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 15px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 2px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 2px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 2px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 2px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image/image.qrc"/>
 </resources>
 <connections/>
</ui>
